vispy/visuals/tests/test_ellipse.py

```diff
@@ -19,20 +19,20 @@ def test_circle_draw():
         ellipse = visuals.Ellipse(pos=(75, 35, 0), radius=20,
                                   color=(1, 0, 0, 1))
         c.draw_visual(ellipse)
-        assert_image_equal("screenshot", 'visuals/circle1.png')
+        assert_image_approved("screenshot", 'visuals/circle1.png')
```

Comment:
...also, once all comments are addressed I'll need to run tests in HiDPI mode to make sure these are all still okay, given that the matching algorithm has changed from xcorr to pixel counting.